lib/util/source-code.js

```diff
@@ -88,68 +88,66 @@ function looksLikeExport(astNode) {
  * @param {ASTNode} ast - The Program node of the AST representing the code. This AST should be created from the text that BOM was stripped.
  * @constructor
  */
-function SourceCode(text, ast) {
-    validate(ast);
+class SourceCode {
```

Comment:
@nzakas It makes sense that we can't make changes to the exported API without a semver-major release. What about purely internal files like Config, ConfigOps, etc.? Can those be done in semver-patch if they're purely internal?